clang/test/SemaSYCL/intel-fpga-local.cpp

```diff
@@ -53,7 +53,22 @@ void foo1()
   //CHECK: IntelFPGANumBanksAttr
   //CHECK-NEXT: ConstantExpr
   //CHECK-NEXT: IntegerLiteral{{.*}}4{{$}}
-  __attribute__((__numbanks__(4))) unsigned int v_six2[32];
+  [[intelfpga::numbanks(4)]] unsigned int v_six2[32];
+
+  //CHECK: VarDecl{{.*}}v_seven
+  //CHECK: IntelFPGAMemoryAttr{{.*}}Implicit
+  //CHECK: IntelFPGAMaxConcurrencyAttr
+  //CHECK-NEXT: ConstantExpr
+  //CHECK-NEXT: IntegerLiteral{{.*}}4{{$}}
+  __attribute__((max_concurrency(4)))
```

Comment:
I am a little bit scared about naming conflicts with thousands of top-level attributes landing from many vendors...

Reply:
This is a good point. We'll think on how to solve this for C-style attributes.